pyrevitlib/pyrevit/revit/db/__init__.py

```diff
@@ -181,23 +181,21 @@ def __init__(self, param_def, param_binding=None, param_ext_def=False):
         # Revit <2017 does not have the Id parameter
         self.param_id = getattr(self.param_def, 'Id', None)
 
-
-        if HOST_APP.is_newer_than(2022, or_equal=True):
-            # GetSpecTypeId() Removed in Revit 2022
-            self.unit_type = self.param_def.GetDataType()
-        elif HOST_APP.is_exactly(2021):
+        if HOST_APP.is_exactly(2021):
```

Comment:
this if branch doesn't set the param_type and param_group attributes

Reply:
I was to prompt to make this PR on my laptop yesterday night, forgot the last two ocmmit before merging! thnaks for the catch